examples/hello.rs

```diff
@@ -94,10 +94,11 @@ fn main() -> Result<(), Box<dyn Error>> {
                 );
 
                 // Get the next frame
-                let frame = swap_chain
-                    .get_current_frame()
-                    .expect("Get next frame")
-                    .output;
+                let frame =
+                    surface.get_current_frame().expect("Get next frame").output;
+                let view = &frame
+                    .texture
+                    .create_view(&wgpu::TextureViewDescriptor::default());
```

Comment:
Is this the proper way to render now? Do we have to explicitly create a view for each frame texture or is there a better way?

Reply:
Yeah seems like it.
Its what all the examples are doing:
* https://github.com/gfx-rs/wgpu/blob/564330432f60b967a69b36a0094d72e1e77ebb72/wgpu/examples/framework.rs#L302
* https://github.com/gfx-rs/wgpu/blob/564330432f60b967a69b36a0094d72e1e77ebb72/wgpu/examples/hello-triangle/main.rs#L101

Reply:
Great!